sktime/markovprocess/_dtraj_stats.py

```diff
@@ -288,6 +285,8 @@ def effective_count_matrix(self):
 
 class TransitionCountEstimator(Estimator):
 
+    # TODO: ctor?
```

Comment:
i think `dt_traj` could be a ctor argument and should potentially be part of the model... :eyes:

Reply:
it is part of the model, but passed on later. The separation/passing around of model/estimator parameters is a bit of a mess across the different estimators.